script/DeployPermissionSplitterProxy.s.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { Script } from "forge-std/Script.sol";
+import { Test } from "forge-std/Test.sol";
+
+import { PermissionSplitterProxy } from "../contracts/PermissionSplitterProxy.sol";
+
+// How to run:
+// 1. `source .env` where `.env has MNEMONIC="x x x ... x" and ETHERSCAN_API_KEY="x" entries
+// 2. forge script script/DeployPermissionSplitterProxy.s.sol:DeployPermissionSplitterProxy --rpc-url $NODE_URL_1-vvvv
+// 3. Verify the above works in simulation mode.
+// 4. Deploy on mainnet by adding --broadcast --verify flags.
+// 5. forge script script/DeployPermissionSplitterProxy.s.sol:DeployPermissionSplitterProxy --rpc-url $NODE_URL_1 --broadcast --verify -vvvv
+contract DeployPermissionSplitterProxy is Script, Test {
+    PermissionSplitterProxy permissionSplitter;
+
+    address constant defaultAdmin = 0xB524735356985D2f267FA010D681f061DfF03715;
+    address constant hubPool = 0xc186fA914353c44b2E33eBE05f21846F1048bEda;
```

Comment:
Does foundry support shared constants in a similar way to how we handle the hardhat deployments now? I assume the constants would be a solidity file

Reply:
are you proposing moving these to a separate file? I kind of think its overkill

Reply:
we will use foundry in all new repos but changing this one over would be pretty difficult because of all the unit tests.

At this point, foundry support is available within the repo so we can use it for new scripts/tests, etc.